extensions/bitwarden/src/search.tsx

```diff
@@ -97,12 +97,33 @@ export function ItemList(props: { api: Bitwarden }) {
   async function refreshItems() {
     if (session.token) {
       const toast = await showToast(Toast.Style.Animated, "Syncing Items...");
-      await bitwardenApi.sync(session.token);
-      await loadItems(session.token);
-      await toast.hide();
+      try {
+        await bitwardenApi.sync(session.token);
+        await loadItems(session.token);
+        await toast.hide();
+      } catch (error) {
+        await bitwardenApi.logout();
+        await session.deleteToken();
+        toast.style = Toast.Style.Failure;
```

Comment:
Will this update the toast ? I did not know it was possible.

Reply:
Yep!